spec/domain/audits/serialize_filter_to_query_parameters_spec.rb

```diff
@@ -0,0 +1,39 @@
+module Audits
+  RSpec.describe SerializeFilterToQueryParameters do
+    let(:allocated_to) { double }
+    let(:audit_status) { double }
+    let(:document_type) { double }
+    let(:organisations) { double }
+    let(:primary_org_only) { double }
+    let(:sort_by) { double }
+    let(:title) { double }
+
+    let(:filter) {
```

Comment:
This is a matter of personal preference, but this test basically checks that we take a hash and reproduce exactly the same hash, which doesn't seem hugely valuable. I know it adds some coupling, but it might be nice to actually instantiate a `Filter` object and pass that in. I think the coupling is okay, because this class' sole purpose in life is to operate on `Filter` objects (as pointed out by its name).

Reply:
I used a double in the spirit of duck typing. We're testing that when given an object with a suitable interface we return a suitable hash. We test the integration elsewhere.